app/forms/draft_collection_form.rb

```diff
@@ -6,6 +6,11 @@ class DraftCollectionForm < Reform::Form
   extend T::Sig
   feature EmbargoDate
 
+  EMBARGO_RELEASE_DURATION_OPTIONS = [['6 months from date of deposit', '6 month'],
```

Comment:
Not sure if it matters, because apparently it used to vary, but should the value here be `6 month` or `6 months`?

Reply:
I just kept the current value from the old code.  Is there a specification somewhere showing that it should be '6 months' and not '6 month' ?